workbench-core/authorization/src/dynamicAuthorization/wbcGroupManagementPlugin.ts

```diff
@@ -101,8 +101,20 @@ export class WBCGroupManagementPlugin implements GroupManagementPlugin {
   ): Promise<IsUserAssignedToGroupResponse> {
     throw new Error('Method not implemented.');
   }
-  public removeUserFromGroup(request: RemoveUserFromGroupRequest): Promise<RemoveUserFromGroupResponse> {
-    throw new Error('Method not implemented.');
+  public async removeUserFromGroup(
+    request: RemoveUserFromGroupRequest
+  ): Promise<RemoveUserFromGroupResponse> {
+    const { groupId, userId } = request;
+
+    try {
+      await this._userManagementService.removeUserFromRole(userId, groupId);
```

Comment:
might be good to note the RPS limit on removeUserFromRole here

Reply:
added `TooManyRequestsError` to all cognito functions